mm2src/common/time_cache.rs

```diff
@@ -27,17 +27,17 @@ use std::collections::VecDeque;
 use std::time::Duration;
 use wasm_timer::Instant;
 
-struct ExpiringElement<Element> {
+pub struct ExpiringElement<Element> {
     /// The element that expires
-    element: Element,
+    pub element: Element,
     /// The expire time.
     expires: Instant,
 }
 
 pub struct TimeCache<Key, Value> {
     /// Mapping a key to its value together with its latest expire time (can be updated through
     /// reinserts).
-    map: FnvHashMap<Key, ExpiringElement<Value>>,
+    pub map: FnvHashMap<Key, ExpiringElement<Value>>,
```

Comment:
As I can see, it has to be public only for `lp_ordermatch::history_committed_changes`. It's worth adding `fn iter()` to TimeCache instead and public getter to `ExpiringElement`.